<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="voteYesButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>96</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>TAK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voteNoButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>96</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>NIE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Zagłosuj nad nową rozgrywką. Czy chcesz zagrać ponownie?</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="winLoseLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>WYGRŁEŚ!!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
